### ISSUE TITLE
Negative energies in iwatch

### ISSUE THREAD

**Issue by rtownson** [nrc-cnrc/EGSnrc#101]
`iwatch` shows Auger electrons with apparent negative energies after Moller. Sample `egslog` and `egsinp` files available upon request.

The following block is the culprit in the `iwatch` routine:
```
ELSEIF(IARG = 9)[
  "surely this logic not needed?"
  IF(NP=NPold)[
     $CNTOUT(NP);(T11,'Interaction rejected',T36,':');
  ]
  ELSE[
     DO IP=NPold,NP[
        KE = E(IP) - ABS(IQ(NP))*RM;
        IF(IP=NPold)[$CNTOUT(IP);(T11,'Resulting electrons',T36,':');]
        ELSE[$CNTOUT(IP);(T36,':');]
     ]
  ]
]
```
should be as follows, which at least doesn't list a photon as an electron with negative energy:
```
ELSEIF(IARG = 9)[
  "surely this logic not needed?"
  IF(NP=NPold)[
     $CNTOUT(NP);(T11,'Interaction rejected',T36,':');
  ]
  ELSE[
     DO IP=NPold,NP[
        KE = E(IP) - ABS(IQ(NP))*RM;
        IF(IP=NPold)[$CNTOUT(IP);(T11,'Resulting electrons',T36,':');]
        ELSE[IF(IQ(IP)=-1)[$CNTOUT(IP);(T36,':');]]
     ]
  ]
]
```

Perhaps a correction to PIRS701 is also called for since it refers to all particles as the relaxation events, which strictly speaking `NPold+1` is not.

(Reported by Dave Rogers)

**Comment by crcrewso** [comment]
Would this trigger negative usteps in the middle of a simple simulation volume?

**Comment by ftessier** [comment]
No this is stricly a "display" bug in the `IWATCH` routine.

**Comment by blakewalters** [comment]
Hi Reid:

Could you send me the offending .egsinp and .egslog files when you get the chance?  Thanks!

**Comment by ftessier** [comment]
Fixed in ba7aca3.
